Source/tmsg.cpp

```diff
@@ -6,44 +6,45 @@ static TMsg *sgpTimedMsgHead;
 
 int __fastcall tmsg_get(BYTE *pbMsg, DWORD dwMaxLen)
 {
-	int len;
-	TMsg *head;
-
-	if ( !sgpTimedMsgHead )
-		return 0;
-
-	if ( (int)(sgpTimedMsgHead->hdr.dwTime - GetTickCount()) >= 0 )
-		return 0;
-	head = sgpTimedMsgHead;
-	sgpTimedMsgHead = head->hdr.pNext;
-	len = head->hdr.bLen;
-	// BUGFIX: ignores dwMaxLen
-	memcpy(pbMsg, head->body, len);
-	mem_free_dbg(head);
-	return len;
+    int len;
+    TMsg *head;
+
+    if (!sgpTimedMsgHead)
+        return 0;
+
+    if ((int)(sgpTimedMsgHead->hdr.dwTime - GetTickCount()) >= 0)
+        return 0;
+    head = sgpTimedMsgHead;
+    sgpTimedMsgHead = head->hdr.pNext;
+    len = head->hdr.bLen;
+    // BUGFIX: ignores dwMaxLen
+    memcpy(pbMsg, head->body, len);
+    mem_free_dbg(head);
+    return len;
 }
 
 void __fastcall tmsg_add(BYTE *pbMsg, BYTE bLen)
 {
-	TMsg **tail;
-
-	TMsg *msg = (TMsg*)DiabloAllocPtr(bLen + sizeof(*msg));
-	msg->hdr.pNext = NULL;
-	msg->hdr.dwTime = GetTickCount() + 500;
-	msg->hdr.bLen = bLen;
-	memcpy(msg->body, pbMsg, bLen);
-	for ( tail = &sgpTimedMsgHead; *tail; tail = &(*tail)->hdr.pNext );
-	*tail = msg;
+    TMsg **tail;
+
+    TMsg *msg = (TMsg *)DiabloAllocPtr(bLen + sizeof(*msg));
+    msg->hdr.pNext = NULL;
+    msg->hdr.dwTime = GetTickCount() + 500;
+    msg->hdr.bLen = bLen;
+    memcpy(msg->body, pbMsg, bLen);
+    for (tail = &sgpTimedMsgHead; *tail; tail = &(*tail)->hdr.pNext)
+        ;

```

Comment:
this looks odd

Reply:
Agreed. Should we enforce braces?

Then this would become.

```c
    for (tail = &sgpTimedMsgHead; *tail; tail = &(*tail)->hdr.pNext) {
        ;
    }
`

Reply:
better i guess

Reply:
I mean, it's strange code to begin with. Having an empty for loop. So this is about the best we can aim for :)

Reply:
Fixed in https://github.com/diasurgical/devilution/pull/390/commits/e7c7e8de46cdb3b829f4e9b05d13e48b4c5cdf2b